code/game/objects/items/weapons/dna_injector.dm

```diff
@@ -142,239 +143,239 @@
 	datatype = DNA2_BUF_SE
 	value = 0xFFF
 
-/obj/item/weapon/dnainjector/hulkmut/New()
+/obj/item/weapon/dnainjector/hulkmut/Initialize()
 	block = HULKBLOCK
-	..()
+	. = ..()
 
 /obj/item/weapon/dnainjector/antihulk
 	name = "\improper DNA injector (Anti-Hulk)"
 	desc = "Cures green skin."
 	datatype = DNA2_BUF_SE
 	value = 0x001
 
-/obj/item/weapon/dnainjector/antihulk/New()
+/obj/item/weapon/dnainjector/antihulk/Initialize()
 	block = HULKBLOCK
-	..()
+	. = ..()

```

Comment:
These variables are set at runtime, that you can't just define them on the class definition, aren't they?

Reply:
They're nonconstant, despite looking like consts. DNA code fills me with pain.